config/locales/en.yml

```diff
@@ -44,6 +44,7 @@ en:
       signed_up_but_unconfirmed: "A message with a confirmation link has been sent to your email address. Please follow the link to activate your account."
       update_needs_confirmation: "You updated your account successfully, but we need to verify your new email address. Please check your email and follow the confirm link to confirm your new email address."
       updated: "Your account has been updated successfully."
+      updated_not_sign_in: "Your password has been changed successfully. please try signing in"
```

Comment:
Some considerations here:
- WDYT about `updated_but_not_signed_in` as the key name?
- Change the default message to `Your account has been updated successfully, but since your password was changed, you need to sign in again`